### ISSUE TITLE
Expiry icon - low contrast/poor visibility

### ISSUE THREAD

**Issue by Githubuxer** [freeotp/freeotp-ios#183]
With the latest update, the expiry icon graphic is not as visible over the icon.

Maybe it should be bright white against the background for improved accessibility/visibility?

**Comment by Githubuxer** [comment]
![image](https://user-images.githubusercontent.com/62911518/86917000-6b1e7c00-c0f2-11ea-8a97-00ffcab7c819.jpeg)
